### DISCUSSION TITLE
normal maps

### DISCUSSION THREAD

**Discussion by graemeniedermayer** [thygate/stable-diffusion-webui-depthmap-script#33] [Ideas]
I got basic normal maps working with a fork of this repo. I was wondering if normal maps are in-scope or out-of-scope of this repo?

Right now it's a very simple code addition, but it might become more complex in the future.

**Comment by thygate** [comment]
I knew about the possibility, but didn't get much further yet than googling the conversion. It's another useful feature for sd.

Great that you got this working, it's amazing how fast this community moves.

Do you want to develop a separate extension, or would you like to colab on this one ?

It would be a great feature to have all in one extension, I will be adding a tab for (3d) viewing in the future.
I'd be happy to add you as colab if you want to maintain the normal map conversion, or you can of course develop your own extension.

**Reply by graemeniedermayer** [reply]
Yah, I'm not entirely sure. I'm still experimenting with some of the filtering aspects so maybe in the near term I'll develop my own extension, then we can think about folding together in the future.

> It would be a great feature to have all in one extension

I definitely agree with this. Maybe a more generic mapping extension that ties all the different mappings together would make sense.

**Reply by thygate** [reply]
Yes, It might be more convenient to keep it separate while developing and testing. Just leave a note if you want to colab.
